src/core/src/Gsim.cc

```diff
@@ -555,23 +554,24 @@ void Gsim::MakeEvent(const G4Event *g4ev, DS::Root *ds) {
     rat_mcpmt->SetType(fPMTInfo->GetType(a_pmt->GetID()));
 
     numPE += a_pmt->GetEntries();
-
+    double chargeScale = DS::RunStore::GetCurrentRun()->GetChannelStatus()->GetChargeScaleByChannel(rat_mcpmt->GetID());
     /** Add "real" hits from actual simulated photons */
     for (int i = 0; i < a_pmt->GetEntries(); i++) {
       // Find the optical process responsible
       auto photon = a_pmt->GetPhoton(i);
       std::string process = photon->GetCreatorProcess();
       if (StoreOpticalTrackID) {
-        AddMCPhoton(rat_mcpmt, a_pmt->GetPhoton(i), exinfo, process);
+        AddMCPhoton(rat_mcpmt, a_pmt->GetPhoton(i), exinfo, process, chargeScale);
       } else {
-        AddMCPhoton(rat_mcpmt, a_pmt->GetPhoton(i), NULL, process);
+        AddMCPhoton(rat_mcpmt, a_pmt->GetPhoton(i), NULL, process, chargeScale);
       }
     }
   }
   mc->SetNumPE(numPE);
 }
 
-void Gsim::AddMCPhoton(DS::MCPMT *rat_mcpmt, const GLG4HitPhoton *photon, EventInfo * /*exinfo*/, std::string process) {
+void Gsim::AddMCPhoton(DS::MCPMT *rat_mcpmt, const GLG4HitPhoton *photon, EventInfo * /*exinfo*/, std::string process,
+                       double chargeScale) {
```

Comment:
I don't think this should be a new line.

Reply:
Tried changing back to one line, but we changed back by the formatting check...
`Checking formatting...
./src/core/src/Gsim.cc:573:119: error: code should be clang-formatted [-Wclang-format-violations]
void Gsim::AddMCPhoton(DS::MCPMT *rat_mcpmt, const GLG4HitPhoton *photon, EventInfo * /*exinfo*/, std::string process, double chargeScale) {
`